core-bundle/src/Command/DebugPagesCommand.php

```diff
@@ -86,13 +86,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
             $rows[] = [
                 $type,
-                $config ? $config->getPath() : '*',
-                $config ? $config->getUrlSuffix() : '*',
+                $config ? $config->getPath() ?: '-' : '*',
+                $config ? $config->getUrlSuffix() ?: '-' : '*',
```

Comment:
```suggestion
                ($config && $config->getPath()) ? $config->getPath() : '*',
                ($config && $config->getUrlSuffix()) ? $config->getUrlSuffix() : '*',
```

Reply:
I infer that you prefer to continue showing a `*` for configurations without a path / url suffix (instead of a `-` as for other cells)?